cl/corpus_importer/management/commands/import_tn.py

```diff
@@ -0,0 +1,188 @@
+import itertools
+import json
+import logging
+from glob import glob
+
+from dateutil import parser
+from django.utils.encoding import force_bytes
+
+from cl.lib.command_utils import VerboseCommand
+from cl.lib.crypto import sha1
+from cl.people_db.models import Person
+from cl.scrapers.management.commands.cl_scrape_opinions import (
+    make_objects,
+    save_everything,
+)
+from cl.scrapers.tasks import extract_doc_content
+from cl.search.models import Court, Docket, Opinion
+
+
+def make_item(case):
+    """Make an import item for our save everything function
+
+    :param case: Case data received from the court.
+    :return: Case information.
+    :type: dict
+    """
+
+    judges = case["judge"]
+    first_name = judges.split(" ")[0].title()
+
+    lead_author = Person.objects.get(
+        positions__court_id=case["court"],
+        is_alias_of=None,
+        name_first=first_name,
+    )
+    # Four Panelists exist, one retired end of 2019 and the other joined.
+    panelists = case["judge"]
```

Comment:
I'd put this in the `else` to make it more clear.

Reply:
√